plugins/sprites.py

```diff
@@ -32,26 +41,43 @@ def generate_sprite_url(dex_entry: JsonDict, shiny: bool = False) -> str:
     else:  # Base form
         dex_name = to_id(dex_entry["name"])
 
-    category = "ani-shiny" if shiny else "ani"
-    return f"https://play.pokemonshowdown.com/sprites/{category}/{dex_name}.gif"
+    ext = "gif" if category.endswith("ani") else "png"
+
+    if back:
+        category += "-back"
+    if shiny:
+        category += "-shiny"
+
+    return f"https://play.pokemonshowdown.com/sprites/{category}/{dex_name}.{ext}"
+
+
+def get_sprite_parameters(args: list[str]) -> tuple[bool, bool, str]:
+    back = False
+    shiny = False
+    category = "ani"
+
+    for arg in args:
+        arg = to_id(arg)
```

Comment:
```suggestion
        arg = arg.lower()
```
We don't really need to remove special characters because we only want to check for equality with `back` or `shiny`.
Trailing whitespaces are already stripped by the `msg.args` property itself.

Reply:
Or is it for compatibility with possible future category aliases?